Sources/Baggage/BaggageContext.swift

```diff
@@ -45,32 +45,24 @@
 ///       }
 ///     }
 public struct BaggageContext: BaggageContextProtocol {
-    private var _storage = [AnyBaggageContextKey: ValueContainer]()
+    private var _storage = [AnyBaggageContextKey: Any]()
```

Comment:
I removed `ValueContainer` to make the implementation a bit more concise.

Reply:
I see, will sanity check if we actually needed it for something but you may be right. Will get to this later today

Reply:
Okay, I'll leave it as `Any` for now but am open to adding `ValueContainer` back in if there are advantages to using it.